manifests/run.pp

```diff
@@ -385,18 +387,24 @@
 
   if $restart {
     if $ensure == 'absent' {
+      $restart_stop_command = [$docker_command, 'stop', "--time=${stop_wait_time}", $sanitised_title]
```

Comment:
`"--time=${stop_wait_time}"`

This will need addressing.

Does the flag actually need the `=`? if not you could just have a clean break between `--time` and `$stop_wait_time`.

Reply:
Tried to keep the commands as untouched as I could. After all, this is not a performance update or code cleanup PR. I can still try to modify it if you think its worth the effort.

Reply:
I think it's worth the effort yes.

The suggestion there is simply suggestion a possible way to make it work without interpolation.

Reply:
Placing a space instead of a = works with no errors. Will try to implement late proper segmentation.